Solutions/Marain.UserNotifications.Storage.AzureStorage/Marain.UserNotifications.Storage.AzureStorage.csproj

```diff
@@ -15,11 +15,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Corvus.Azure.Storage.Tenancy" Version="2.0.7" />
-    <PackageReference Include="Endjin.RecommendedPractices" Version="1.2.0">
+    <PackageReference Include="Corvus.Storage.Azure.TableStorage.Tenancy" Version="3.2.0-PullRequest0409.5" />
+    <PackageReference Include="Corvus.Storage.Azure.BlobStorage.Tenancy" Version="3.2.0-PullRequest0409.5" />
```

Comment:
So we can't actually merge this PR until non-preview versions of these are available.

Have you done any testing beyond running the automated test suite? Since your PR is the first ever piece of code to attempt to use the new `Corvus.Storage`/`Corvus.Tenancy` support for Azure Tables, I want to make sure we've actually seen `Marain.UserNotifications` exercising the new functionality before declaring that that the changes in Corvus are good to go from preview to release.